README.md

```diff
@@ -93,14 +93,8 @@ The following is the minimum needed code to send an email. You may find more exa
 
 ```php
 <?php
-require 'vendor/autoload.php'; // If you're using Composer (recommended)
-// Comment out the above line if not using Composer
-// require("<PATH TO>/sendgrid-php.php");
-// If not using Composer, uncomment the above line and
-// download sendgrid-php.zip from the latest release here,
-// replacing <PATH TO> with the path to the sendgrid-php.php file,
-// which is included in the download:
-// https://github.com/sendgrid/sendgrid-php/releases
+// Uncomment next line if you're not using a dependency loader (such as Composer)
```

Comment:
First, a bit of context. We had a period where were getting a steady stream of issues at our support channel with confusion on installation, which virtually went away with this updated content. So I'm nervous about removing it entirely, but I do agree it's still verbose and likely not necessary for the majority of use cases.

Above this line, I think we should still include:
`require 'vendor/autoload.php'; // If you're using Composer (recommended)`

Then, something like:
`// Comment out the above line and uncomment the next line if you're not using a dependency loader (such as Composer)`

And for this part:
```
// download sendgrid-php.zip from the latest release here,	
// replacing <PATH TO> with the path to the sendgrid-php.php file,	
// which is included in the download:	
// https://github.com/sendgrid/sendgrid-php/releases
```
we can move those instructions to https://github.com/sendgrid/sendgrid-php/blob/master/README.md#install-package where we can then link from here.

Reply:
I went ahead and made the changes.